docs/data-sources/tenants.md

```diff
@@ -18,31 +18,28 @@ Provides information about existing tenants.
 ### Optional
 
 - `cloned_from_tenant_id` (String) A filter to search for a cloned tenant by its ID.
+- `id` (String) The unique ID for this resource.
 - `ids` (List of String) A filter to search by a list of IDs.
 - `is_clone` (Boolean) A filter to search for cloned resources.
 - `name` (String) A filter to search by name.
-- `partial_name` (String) A filter to search by the partial match of a name.
+- `partial_name` (String) A filter to search by a partial name.
 - `project_id` (String) A filter to search by a project ID.
 - `skip` (Number) A filter to specify the number of items to skip in the response.
-- `space_id` (String) A Space ID to filter by. Will revert what is specified on the provider if not set.
+- `space_id` (String) The space ID associated with this tenants.
 - `tags` (List of String) A filter to search by a list of tags.
 - `take` (Number) A filter to specify the number of items to take (or return) in the response.
-
-### Read-Only
-
-- `id` (String) An auto-generated identifier that includes the timestamp when this data source was last modified.
 - `tenants` (Block List) A list of tenants that match the filter(s). (see [below for nested schema](#nestedblock--tenants))
 
 <a id="nestedblock--tenants"></a>
 ### Nested Schema for `tenants`
 
-Read-Only:
+Optional:
```

Comment:
These were all read-only previously...

Reply:
Do you know why these change and how to keep them as read only?